caveclient/skeletonservice.py

```diff
@@ -365,3 +482,105 @@ def get_skeleton(
             return skeleton_bytesio
 
         raise ValueError(f"Unknown output format: {output_format}")
+
+    def get_bulk_skeletons(
```

Comment:
I would not call this "get" since it doesn't return skeletons, but rather, trigger_bulk_skeleton_generation or something.

Reply:
This returns skeletons. The other new API function doesn't, but this one does.